File: src/main/java/commands/slashadapters/adapters/CooldownsAdapter.java
package commands.slashadapters.adapters;

import commands.runnables.fisherycategory.CooldownsCommand;
import commands.slashadapters.Slash;
import commands.slashadapters.SlashAdapter;
import commands.slashadapters.SlashMeta;
import net.dv8tion.jda.api.events.interaction.SlashCommandEvent;
import net.dv8tion.jda.api.interactions.commands.build.CommandData;

@Slash(command = CooldownsCommand.class)
public class CooldownsAdapter extends SlashAdapter {

    public CommandData addOptions(CommandData commandData) {
        return commandData;
    }

    @Override
    public SlashMeta process(SlashCommandEvent event) {
        return new SlashMeta(CooldownsCommand.class, "");
    }

}

File: src/main/java/commands/slashadapters/adapters/CooldownsAdapter.kt
package commands.slashadapters.adapters

import commands.runnables.fisherycategory.CooldownsCommand
import commands.slashadapters.Slash
import commands.slashadapters.SlashAdapter
import commands.slashadapters.SlashMeta
import mysql.hibernate.entity.GuildEntity
import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent
import net.dv8tion.jda.api.interactions.commands.build.SlashCommandData

@Slash(command = CooldownsCommand::class)
class CooldownsAdapter : SlashAdapter() {

    public override fun addOptions(commandData: SlashCommandData): SlashCommandData {
        return commandData
    }

    override fun process(event: SlashCommandInteractionEvent, guildEntity: GuildEntity): SlashMeta {
        return SlashMeta(CooldownsCommand::class.java, "")
    }

}